tico/serialize/circle_mapping.py

```diff
@@ -134,16 +134,49 @@ def extract_circle_shape(node: torch.fx.Node) -> Tuple[List[int], Optional[List[
 
 def to_circle_shape(torch_shape: torch.Size) -> Tuple[List[int], Optional[List[int]]]:
     shape: List[int] = list(torch_shape)
-    shapeSignature: Optional[List[int]] = None
+    shape_signature: Optional[List[int]] = None
 
     if any(isinstance(s, torch.SymInt) for s in shape):
-        shapeSignature = shape.copy()
+        shape_signature = shape.copy()
         for idx, s in enumerate(shape):
             if isinstance(s, torch.SymInt):
                 shape[idx] = 1
-                shapeSignature[idx] = -1
+                shape_signature[idx] = -1
 
-    return shape, shapeSignature
+    return shape, shape_signature
+
+
+def validate_circle_shape(shape: List[int], shape_signature: Optional[List[int]]):
```

Comment:
This would improve the validity of the generated circle. 👍 

Could you make some tests for this functions?